templates/aws/eb/HA/ws/docker-compose.yml.dist

```diff
@@ -0,0 +1,48 @@
+version: '3.8'
+
+services:
+    gateway:
+        container_name: '${GATEWAY_SERVICE_NAME}'
+        image: nginx:1.19.6-alpine
+        restart: always
+        depends_on:
+            - supportpal_cron
+        volumes:
+            - ./gateway/nginx-http.conf:/etc/nginx/templates/default.conf.template
+            - supportpal_resources:/supportpal/resources:ro
+            - supportpal_app:/supportpal/app:ro
+        environment:
+            WEB_CONTAINER: ${CRON_SERVICE_NAME}
+        ports:
+            - '80:80'
```

Comment:
Need to account for SSL - https://beyondco.de/docs/laravel-websockets/basic-usage/ssl

Reply:
SSL is on the level of load balancer, which would always contact port 80